src/lib/util/Config.ts

```diff
@@ -25,7 +26,8 @@ const DEFAULT_CONFIG: ConfigContent = {
 			targetPatterns: [
 				"**/*.cls","**/*.java","**/*.js","**/*.page","**/*.component","**/*.xml",
 				"!**/node_modules/**","!**/*-meta.xml"
-			]
+			],
+			supportedLanguages: ['apex', 'javascript']
```

Comment:
These values are used in multiple places, would be good to move to Constants.ts

Reply:
+1